modules/jumpstart_ui/src/Plugin/paragraphs/Behavior/HeroPatternBehavior.php

```diff
@@ -101,6 +129,7 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
   public function view(array &$build, ParagraphInterface $paragraph, EntityViewDisplayInterface $display, $view_mode) {
     // FYI: this adds the class one level above than the pattern template.
     $build['#attributes']['class'][] = 'overlay-' . $paragraph->getBehaviorSetting('hero_pattern', 'overlay_position', 'left');
+    $build['#attributes']['class'][] = 'overlay-color-' . $paragraph->getBehaviorSetting('hero_pattern', 'overlay_color', 'default');
+    $build['#attributes']['class'][] = 'bottom-margin-' . $paragraph->getBehaviorSetting('hero_pattern', 'space_below', 'default');
```

Comment:
instead of `default`, maybe wrap it in an `if` statement. Something like this:
```suggestion
    if ($color = $paragraph->getBehaviorSetting('hero_pattern', 'overlay_color')) {
      $build['#attributes']['class'][] = 'overlay-color-' . $color;
    }
    $build['#attributes']['class'][] = 'bottom-margin-' . $paragraph->getBehaviorSetting('hero_pattern', 'space_below', 'default');
```